.github/workflows/test.yml

```diff
@@ -201,96 +203,46 @@ jobs:
       OVN_GATEWAY_MODE: "${{ matrix.gateway-mode }}"
     steps:
 
-    - name: Free up disk space
-      run: sudo eatmydata apt-get remove --auto-remove -y aspnetcore-* dotnet-* libmono-* mono-* msbuild php-* php7* ghc-* zulu-*
-
-    - name: Set up Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: ${{ env.GO_VERSION }}
-      id: go
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-
-    - name: Set up environment
-      run: |
-        export GOPATH=$(go env GOPATH)
-        echo "GOPATH=$GOPATH" >> $GITHUB_ENV
-        echo "$GOPATH/bin" >> $GITHUB_PATH
-
-    - name: Disable ufw
-      # For IPv6 and Dualstack, ufw (Uncomplicated Firewall) should be disabled.
-      # Not needed for KIND deployments, so just disable all the time.
-      run: |
-        sudo ufw disable
-
-    - name: Restore Kubernetes from cache
-      id: cache-k8s
-      uses: actions/cache@v2
-      with:
-        path: "${{ env.GOPATH }}/src/k8s.io/kubernetes/"
-        key: k8s-go-2-${{ env.K8S_VERSION }}
-
-    # Re-build if kube wasn't in the cache due to
-    # https://github.com/actions/cache/issues/107#issuecomment-598188250
-    # https://github.com/actions/cache/issues/208#issuecomment-596664572
-    - name: Build and install Kubernetes
-      if: steps.cache-k8s.outputs.cache-hit != 'true'
-      run: |
-        set -x
-        rm -rf $GOPATH/src/k8s.io/kubernetes/
-        git clone --single-branch --branch $K8S_VERSION https://github.com/kubernetes/kubernetes.git $GOPATH/src/k8s.io/kubernetes/
-        pushd $GOPATH/src/k8s.io/kubernetes/
-        make WHAT="test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cmd/kubectl"
-        rm -rf .git
-
-    - uses: actions/download-artifact@v2
-      with:
-        name: test-image
-    - name: Load docker image
-      run: |
-        docker load --input image.tar
-
-    - name: kind setup
-      run: |
-        export OVN_IMAGE="ovn-daemonset-f:dev"
-        make -C test install-kind
+      - name: Run e2e tests
+        uses: jluhrsen/ovn-kubernetes/.github/actions/common@master
```

Comment:
Do actions receive environment variables?

Reply:
oh... ignore this commit. I didn't realize I was pushing it to the same PR. I'm still trying to figure this part out in my fork.

Reply:
ok, PR is updated. please review now. Once I have the de-dup stuff working in my fork, I'll push another PR for that.